<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>969</width>
    <height>578</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>I2C BUS ACCESS</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="frm_I2CBusStatus">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>331</width>
      <height>271</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="lineWidth">
     <number>7</number>
    </property>
    <property name="midLineWidth">
     <number>1</number>
    </property>
    <widget class="QLabel" name="lbl_I2CStatus">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>261</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(85, 85, 255);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="lineWidth">
      <number>4</number>
     </property>
     <property name="text">
      <string>I2C TASK STATE</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_I2CStandBy">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>90</y>
       <width>261</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 170, 255);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="lineWidth">
      <number>4</number>
     </property>
     <property name="text">
      <string>STAND BY I2C TASK</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_I2CInUse">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>140</y>
       <width>261</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(0, 170, 0);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="lineWidth">
      <number>4</number>
     </property>
     <property name="text">
      <string>RUN BASIC I2C TASK</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_I2CInConflict">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>190</y>
       <width>261</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 0, 0);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="lineWidth">
      <number>4</number>
     </property>
     <property name="text">
      <string>CONFLICT</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frm_I2CSimButton">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>0</y>
      <width>181</width>
      <height>161</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="lineWidth">
     <number>4</number>
    </property>
    <widget class="QPushButton" name="btn_RunBus">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>161</width>
       <height>28</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Sans Serif</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(85, 255, 127);</string>
     </property>
     <property name="text">
      <string>RUN MAIN I2C TASK</string>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_StopBus">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>161</width>
       <height>28</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Sans Serif</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(85, 255, 127);</string>
     </property>
     <property name="text">
      <string>STOP MAIN I2C TASK</string>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frm_I2CWriteStatus">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>0</y>
      <width>331</width>
      <height>441</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="lineWidth">
     <number>7</number>
    </property>
    <property name="midLineWidth">
     <number>1</number>
    </property>
    <widget class="QLabel" name="lbl_I2CSWriteStatus">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>261</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(85, 85, 255);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="lineWidth">
      <number>4</number>
     </property>
     <property name="text">
      <string>I2C WRITE STATE</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_I2CWriteIdle">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>70</y>
       <width>261</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(0, 170, 0);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="lineWidth">
      <number>4</number>
     </property>
     <property name="text">
      <string>IDLE</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_I2CWriteRequest">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>120</y>
       <width>261</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255,255,255);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="lineWidth">
      <number>4</number>
     </property>
     <property name="text">
      <string>WRITE REQUEST</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_I2CWriteInProgress">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>170</y>
       <width>261</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255,255,255);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="lineWidth">
      <number>4</number>
     </property>
     <property name="text">
      <string>WRITE IN PROGRESS</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_I2CWriteDone">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>220</y>
       <width>261</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255,255,255);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="lineWidth">
      <number>4</number>
     </property>
     <property name="text">
      <string>WRITE DONE</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_I2CWriteFail">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>270</y>
       <width>261</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255,255,255);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="lineWidth">
      <number>4</number>
     </property>
     <property name="text">
      <string>WRITE FAIL</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_I2CWriteBusFail">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>320</y>
       <width>261</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255,255,255);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="lineWidth">
      <number>4</number>
     </property>
     <property name="text">
      <string>BUS FAIL</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_I2CWriteEnd">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>370</y>
       <width>261</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255,255,255);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="lineWidth">
      <number>4</number>
     </property>
     <property name="text">
      <string>WRITE END</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="btn_StartTask_1">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>360</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>START TASK 1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_StartTask_2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>360</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>START TASK 2</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>969</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
